### DISCUSSION TITLE
How to remove the rating info from the screen

### DISCUSSION THREAD

**Discussion by Logansfury** [nomacs/nomacs#1035] [General]
Hello, I installed Nomacs v3.12 from the Software Manager on my Linux Mint box. I cannot find anything relating to the popup in the bottom right corner of the screen where the image is displayed where a star rating system is displayed. How can I get rid of this please?

**Comment by leejuyuu** [comment]
Do you mean this?
![image](https://github.com/nomacs/nomacs/assets/46868116/8ebce0a6-e818-4ffe-a3c4-2d6b0db36078)

Toggle its appearance from `Panels > File Info` or use the keyboard shortcut `i`.

**Comment by Logansfury** [comment]
Hello and thank you for the reply.

Yes your image.png is exactly the window I was talking about. I found a
"Hide All" option under Panels and checked that, now I have a perfect,
clean, uncluttered, windowless image display as I wanted.
